packages/pangea-sdk/pangea/services/authn/models.py

```diff
@@ -434,281 +287,205 @@ class ClientJWKSResult(PangeaResponseResult):
     keys: List[Union[JWKec, JWKrsa, JWK]]
 
 
-# - path: authn::/v1/flow/complete
-# https://pangea.cloud/docs/api/authn#complete-sign-up-in
-class FlowCompleteRequest(APIRequestModel):
-    flow_id: str
-
-
-class FlowCompleteResult(PangeaResponseResult):
-    refresh_token: LoginToken
-    login_token: LoginToken
-
-
-# - path: authn::/v1/flow/enroll/mfa/complete
-# https://pangea.cloud/docs/api/authn#complete-mfa-enrollment
-class FlowEnrollMFACompleteRequest(APIRequestModel):
-    flow_id: str
-    code: str
-    cancel: Optional[bool] = None
-
-
-class EnrollMFAStart:
-    mfa_providers: List[str]
-
-
-class TOTPsecret:
-    qr_image: str
-    secret: str
-
-
-class EnrollMFAComplete:
-    totp_secret: TOTPsecret
+class UserAuthenticatorsDeleteRequest(APIRequestModel):
+    user_id: str
+    mfa_provider: MFAProvider
 
 
-class SocialSignup:
-    redirect_uri: str
+class UserAuthenticatorsDeleteResult(PangeaResponseResult):
+    pass
 
 
-class PasswordSignup:
-    password_chars_min: int
-    password_chars_max: int
-    password_lower_min: int
-    passwrod_upper_min: int
-    password_punct_min: int
+class UserAuthenticatorsListRequest(APIRequestModel):
+    email: Optional[str] = None
+    id: Optional[str] = None
 
 
-class VerifyCaptcha:
-    site_key: str
+class Authenticator(APIResponseModel):
+    id: str
+    type: str
+    enable: bool
+    provider: Optional[str] = None
+    rpid: Optional[str] = None
+    phase: Optional[str] = None
 
 
-class VerifyMFAStart:
-    mfa_providers: List[str]
+class UserAuthenticatorsListResult(PangeaResponseResult):
+    authenticators: List[Authenticator]
 
 
-class VerifyPassword:
-    password_chars_min: int
-    password_chars_max: int
-    password_lower_min: int
-    passwrod_upper_min: int
-    password_punct_min: int
+class FlowCompleteRequest(APIRequestModel):
+    flow_id: str
 
 
-class Signup:
-    social_signup: SocialSignup
-    password_signup: PasswordSignup
+class FlowCompleteResult(PangeaResponseResult):
+    refresh_token: LoginToken
+    active_token: LoginToken
 
 
-class VerifySocial:
-    redirect_uri: str
+class FlowChoices(APIResponseModel):
```

Comment:
this describes "FlowChoice" but not "FlowChoices".   "FlowChoices" is described as:
```
type Flow2Choices struct {
    ID         string        `json:"flow_id"`
    Type       []string      `json:"flow_type"`
    Email      string        `json:"email,omitempty"`
    Disclaimer string        `json:"disclaimer,omitempty"`
    Phase      string        `json:"flow_phase"`
    Choices    []Flow2Choice `json:"flow_choices"`
}
```

Reply:
Fixed